cdip_connector/core/connector_base.py

```diff
@@ -44,19 +39,21 @@ def execute(self) -> None:
 
     async def main(self) -> None:
         try:
-            # Fudge with a really long timeout value.
-            async with ClientSession(
-                timeout=ClientTimeout(total=CLIENT_TIMEOUT_TOTAL)
-            ) as session:
-                self.logger.info(f"CLIENT_ID: {cdip_settings.KEYCLOAK_CLIENT_ID}")
-                integrations = await self.portal.get_authorized_integrations(session)
-
-                # result = [await self.extract_load(session, i) for i in integration_info]
-                for idx in range(0, len(integrations), self.concurrency):
+
+            async with ClientSession() as s:
```

Comment:
I removed the timeout and simply rely on the default 5 minutes timeout in ClientSession.